### ISSUE TITLE
Resources page > design system icon button has no label

### ISSUE THREAD

**Issue by madcampos** [techtankto/techtank-site#173]
The design system should provide best practices of the patterns used, thus it should be _the_ most accessible page[^1].

The icon button example does not include an accessible label.

<img width="403" height="101" alt="Screenshot of the buttons sizes section highlighting the icon button" src="https://github.com/user-attachments/assets/fe62be53-2979-4a22-8f89-d0409a5ab770" />

[^1]: Exceptions here are things like heading levels, where it would include multiple `h1` tags for demonstration purposes of the components that use those elements.
